src/components/NavBar.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import '../stylesheets/navBar.scss'
+
+export interface Link {
+    text: string;
+    url: string;
+}
+export interface NavigationProps {
+    links: Array<Link>;
+}
+
+export class NavBar extends React.Component<NavigationProps, {}> {
+    render() {
+        return (<nav className="nav-bar">
+                {this.props.links.map((link) => <a className="nav-item" href={link.url}>{link.text}</a>)}
```

Comment:
```suggestion
                {this.props.links.map((link) => <a className="nav-item" key={someKey} href={link.url}>{link.text}</a>)}
```
There is a warning on browser console if we dont't add a key here. Maybe we can use {link.text} as the key?

Reply:
Ah good catch. Thanks, will add that in.